frontend/common/src/helpers/dateUtils.ts

```diff
@@ -10,7 +10,14 @@ import { getLocale, Locales } from "./localize";
 export const DATE_FORMAT_STRING = "yyyy-MM-dd";
 // DateTime scalar formatting string
 export const DATETIME_FORMAT_STRING = "yyyy-MM-dd HH:mm:ss";
-export const LONG_DAY_FORMAT_STRING = "EEEE, d MMMM yyyy";
+
+// https://stackoverflow.com/a/563442/18246559
+const addDays = (startDate: Date, days: number) => {
+  // not really const since setDate changes the value in place
+  const date = new Date(startDate.valueOf());
+  date.setDate(date.getDate() + days);
+  return date;
+};
```

Comment:
Just a tip if we want to reduce code we need to maintain, `date-fns` has an [`add`](https://date-fns.org/v2.29.3/docs/add) helper ;)

```tsx
const result = add(date, { days: 1 });
```

Reply:
Good catch!  I should have guessed that was in there.  Fixed in [Use date-fns dateadd](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4407/commits/728c9240d3174f7b2af21774c9ae93cd921fb248).